### PR TITLE
Tutorial explorer: report tutorial start by grade range

### PR DESCRIPTION
**Pull request by breville**
As well as reporting a tutorial event action `"start"` with event label of tutorial code, report a tutorial event action `"start-[grade]"` with event label of tutorial code (with `[grade]` being `"all"`, `"pre"`, `"2-5"`, `"6-8"` or `"9+"`).

This will give us five new buckets of tutorial starts so that we can see which tutorials are most popular in each grade range.

### CODE REVIEW

File: apps/src/tutorialExplorer/tutorialDetail.jsx [hunk 1]
```diff
@@ -111,7 +111,8 @@ const TutorialDetail = React.createClass({
     closeClicked: PropTypes.func.isRequired,
```
Comment (Erin007) [inline apps/src/tutorialExplorer/tutorialDetail.jsx:111]: Want to update to `extends React.Component` while you're in the file or save for follow up?

File: apps/test/unit/tutorialExplorer/tutorialSetTest.jsx [hunk 2]
```diff
@@ -38,6 +39,7 @@ describe('TutorialSet', () => {
           localeEnglish={true}
           disabledTutorial={false}
           changeTutorial={instance.changeTutorial}
+          grade={"all"}
```
Comment (Erin007) [inline apps/test/unit/tutorialExplorer/tutorialSetTest.jsx]: do you need { } here?

**Comment by islemaster** [comment]
If I'm reading this correctly, the new event name is determined by which grade filters the user currently has applied, rather than the recommended age range of the specific tutorial started.  That makes sense given we should be able to reconstruct the latter from tutorial shortcodes if needed.  Just double-checking that this is by design?

File: apps/src/tutorialExplorer/tutorialDetail.jsx (hunk 1 above)
Comment (breville) [reply apps/src/tutorialExplorer/tutorialDetail.jsx:111]: Good question.  I'd like to do that in a standalone PR, just to isolate concerns.

File: apps/test/unit/tutorialExplorer/tutorialSetTest.jsx (hunk 2 above)
Comment (breville) [reply apps/test/unit/tutorialExplorer/tutorialSetTest.jsx]: Good call.  Fixing.

**Comment by breville** [comment]
@islemaster Yes.  This will give us the data to revive "Sort by popularity" and show the correct order based on the currently-selected grade band.

**Pull request merged**